purchases/src/test/java/com/revenuecat/purchases/common/verification/SigningManagerTest.kt

```diff
@@ -269,8 +333,46 @@ class SigningManagerTest {
         nonce: String = "MTIzNDU2Nzg5MGFi",
         body: String? = "{\"request_date\":\"2023-02-21T18:58:36Z\",\"request_date_ms\":1677005916011,\"subscriber\":{\"entitlements\":{},\"first_seen\":\"2023-02-21T18:58:35Z\",\"last_seen\":\"2023-02-21T18:58:35Z\",\"management_url\":null,\"non_subscriptions\":{},\"original_app_user_id\":\"login\",\"original_application_version\":null,\"original_purchase_date\":null,\"other_purchases\":{},\"subscriptions\":{}}}\n",
         requestTime: String? = "1677005916012",
-        eTag: String? = null
-    ) = signingManager.verifyResponse(requestPath, signature, nonce, body, requestTime, eTag)
+        eTag: String? = null,
+        postParamsHeader: String? = null,
+    ) = signingManager.verifyResponse(requestPath, signature, nonce, body, requestTime, eTag, postParamsHeader)
+
+    // We can use this function to create new signatures if we need to change the test data
+    @Suppress("Unused")
+    private fun createFakeSignature(
```

Comment:
This is a messy implementation to create signatures for the tests. I didn't check it in before, but I think it's useful. I don't think we should use it during the tests though since it has a lot of logic. I prefer to hardcode the expected signature.